_config.yml

```diff
@@ -34,18 +34,21 @@ defaults:
   values:
     layout: post
     permalink: "/analysis/:slug/"
+    view_all_url: "/analysis/"
 - scope:
     path: ''
     type: 'briefs'
   values:
     layout: post
     permalink: "/issue-briefs/:slug/"
+    view_all_url: "/analysis/"
 - scope:
     path: ''
     type: 'workshops'
   values:
     layout: post
     permalink: "/workshops/:slug/"
+    view_all_url: "/workshops/"
```

Comment:
Why do you need to define this `view_all_url` in the config file? Because it's a page variable? (`page.view_all_url`)

Reply:
Correct. The defaults for each collection type set in the config are turned into page variables that can be used in the layouts & includes. Since I always wanted analysis & workshops to have the same view all URL, it made sense to define them in the configuration rather than doing separate if statements in the component itself.